ani-cli

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 # version number
-VERSION="1.5.4"
+VERSION="1.6.0"
```

Comment:
Why all the way to 1.6.0 ?

Reply:
cause that was the milestone set in the issue, if you want I will lower it down

Reply:
Yeah, please do. But honestly we have way to few issues right now

Reply:
Done, you can review